### DISCUSSION TITLE
Release 1.0.0

### DISCUSSION THREAD

**Discussion by feuzeu** [lagdo/tontine#25] [Announcements]
First stable release.

<hr /><em>This discussion was created from the release <a href='https://github.com/lagdo/tontine/releases/tag/v1.0.0'>Release 1.0.0</a>.</em>

**Comment by appdemain** [comment]
Bonjour puissions nous discuter en privé, j'aimerai utiliser votre script mais je rencontre des difficultés pour l'installation, et si possible j'aimerais l'amélioration des fonctionnalités comme un service payant. Merci

**Reply by feuzeu** [reply]
Bonjour,

Pour tout problème technique, il ajouter une entrée dans la section Issues de ce dépôt, en donnant les informations nécessaires pour comprendre pourquoi.
La version de php utilisée, le logiciel de base de données et sa version, l'étape où le problème s'est produit, le message d'erreur obtenu.

**Reply by feuzeu** [reply]
Quant au paiement, il y a tellement de formules possibles qu'on ne peut pas mettre cette fonction directement dans le logiciel open source.
Il existe de nombreux outils tiers qui permettent de gérer ces aspects.

**Reply by appdemain** [reply]
Puissions nous discuter sur whatsapp ?

**Reply by appdemain** [reply]
+221784489010
